frontend/insurance/src/app/shared/components/files/files.component.ts

```diff
@@ -143,13 +155,57 @@ export class FilesComponent implements OnInit {
       this.service
         .addFiles(this.files, this.claim.vehicle.vin, this.fileType)
         .subscribe(data => {
-          console.log(data);
           const claim = <Protos.Claim>data;
           this.updatedClaim.emit(claim);
           this.files = null;
           this.success = true;
           this.fileList = [];
+          this.urls = [];
         });
     }
   }
+
+  onDrop(event: any, fileType: string) {
+    event.preventDefault();
+    const files = <FileList>event.dataTransfer.files;
```

Comment:
Why do you do this? Don't use type casting unless you have to.. This masks errors.

Reply:
Also, using the `any` type argument is for n00bs.